components/interactive_dialog/dialog_element/dialog_element.tsx

```diff
@@ -121,7 +136,7 @@ export default class DialogElement extends React.PureComponent {
             );
         }
 
-        let helpTextContent = helpText;
+        let helpTextContent: React.ReactElement | string | undefined;
```

Comment:
Similarly, that also applies here
```suggestion
        let helpTextContent: React.ReactNode;
```